docker/cheqd_node/node-runner.sh

```diff
@@ -5,16 +5,33 @@
 set -euox pipefail
 
 CHEQD_ROOT_DIR="$HOME/.cheqdnode"
+CHEQD_COMMON_DIR="$HOME/.common"
 
 # Init node config directory
 if [ ! -d "${CHEQD_ROOT_DIR}/config" ]
 then
     echo "Node config not found. Initializing."
-    cheqd-noded init $NODE_MONIKER --home 
+    cheqd-noded init $NODE_MONIKER --home ${CHEQD_ROOT_DIR}
```

Comment:
This should map to the volume path shouldn't it? Otherwise, the volume path and this path could likely in in different places?

Reply:
Well, for now it doesn't work, cause `--home` requires a value after. Actually, for now I'm going to get rid of `--home` parameter at all.